Sources/App/routes.swift

```diff
@@ -73,6 +81,10 @@ func routes(_ app: Application) throws {
     try apiRoutes.grouped("schedule").register(collection: ScheduleAPIController())
     try apiRoutes.grouped("local").register(collection: LocalAPIController())
 
+    let apiTwoRoutes = app.grouped("api", "v2")
```

Comment:
Why is this a v2 route, rather than replacing v1 as the App hasn't been finished yet?

Reply:
I got pushback when I changed a value before during development so felt it best to avoid that here given its an even bigger change and just make the new one.